openqabot/approver.py

```diff
@@ -116,51 +116,51 @@ def _approvable(self, inc: IncReq) -> bool:
         return True
 
     @lru_cache(maxsize=512)
-    def is_job_marked_acceptable_for_incident(self, job: JobAggr, inc: int) -> bool:
+    def is_job_marked_acceptable_for_incident(self, job_id: int, inc: int) -> bool:
         regex = re.compile(r"\@review\:acceptable_for\:incident_%s\:(.+)" % inc)
         try:
-            for comment in self.client.get_job_comments(job.job_id):
+            for comment in self.client.get_job_comments(job_id):
                 if regex.match(comment["text"]):
                     return True
         except RequestError:
             pass
         return False
 
     @lru_cache(maxsize=128)
-    def get_jobs(self, job: JobAggr, api: str, inc: int) -> bool:
+    def get_jobs(self, job_aggr: JobAggr, api: str, inc: int) -> bool:
         results = requests.get(
-            QEM_DASHBOARD + api + str(job.job_id), headers=self.token
+            QEM_DASHBOARD + api + str(job_aggr.id), headers=self.token
         ).json()
 
         # keep jobs explicitly marked as acceptable for this incident by openQA comments
         for res in results:
             ok_job = res["status"] == "passed"
             if ok_job:
                 continue
-            if self.is_job_marked_acceptable_for_incident(job, inc):
+            if self.is_job_marked_acceptable_for_incident(res["job_id"], inc):
                 log.info(
                     "Ignoring failed job %s for incident %s due to openQA comment"
-                    % (job.job_id, inc)
+                    % (res["job_id"], inc)
                 )
                 res["status"] = "passed"
             else:
                 log.info(
-                    "Found failed, not-ignored job %s for incident %s"
-                    % (job.job_id, inc)
+                    "Found failed, not-ignored job setting %s for incident %s"
+                    % (job_aggr.id, inc)
```

Comment:
Can we also use something like `res["job_id"]` here to refer to real openQA job IDs?

Reply:
Ah, yes... in this context we can. I will update the PR.